javascript/javascript1/week2/studentManager.js

```diff
@@ -0,0 +1,35 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+  if (getNumberOfStudents(class07Students) < 6) {
+    if (studentName == " ") {
+      output = "You cannot add an empty string to a class";
+      return output;
+    } else {
+      for (let i = 0; i < getNumberOfStudents(class07Students); i++) {
+        if (studentName == class07Students[i]) {
+          output = "student " +  studentName  + "is already in the class";
+          return output;
+        }
+      }
+    }
+  } else if (studentName == "Queen") {
```

Comment:
what about if `studentName` is `queen` 
To include all cases we can call `toLocaleLowerCase` method on strings
```js
} else if (studentName.toLocaleLowerCase() === "queen") {
```